src/routes/Result.tsx

```diff
@@ -37,14 +45,24 @@ const useStyles = makeStyles(theme => ({
 }));
 
 
-export default function Process(props: any) {
+export default function Result() {
     const classes = useStyles();
+    const [history, setHistory] = React.useState([]);
+
     return (
         <div className={classes.center}>
             <CheckCircleIcon className={classes.icon}/>
             <Typography variant={'h2'}>Created Ciphertext</Typography>
             <Card className={classes.copybox}>
-                <InputBase placeholder={"jaslkdj3ouroqejdnfskf"} value={'jaslkdykjhgjy97yhuhkhikuhkjhkjhkjj3ou'} />
+                <p>Key: </p>
+                <InputBase placeholder={key} value={key} />
```

Comment:
How would I go about getting data into here from DropDownButton.jsx https://github.com/fairhurt/securebin/blob/de932aa5e6f2ade09bd3cd87ae1260e5c9882004/src/routes/DropDownButton.jsx#L157

Reply:
I'm creating a global context right now that will allow you to globally access that state, I need probably a few hours to finish it tho

Reply:
Sounds good